app/tests/spec/views/coppa/coppa-age-input.js

```diff
@@ -67,6 +67,29 @@ define(function (require, exports, module) {
       });
     });
 
+    describe('onKeyDown', function () {
+      it('force digit input', function () {
+        var age = $('#age');
+        age.val('1.1');
+        view.onInput();
+        assert.equal(age.val(), '11');
+      });
+
+      function testEnterKeyTriggersSubmit(element, done) {
+        view.on('submit', function () {
+          done();
+        });
+
+        // submit using the enter key
+        var e = jQuery.Event('keydown', { which: 13 });
```

Comment:
I think this file existed before we had [key-codes.js](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/lib/key-codes.js#L9) and the magic number `13` is a left-over from those days. Between lines 11 and 12, can you add a new line:
```js
var KeyCodes = require('lib/key-codes');
```
Then replace `13` on this line with `KeyCodes.ENTER`.

I'm undecided whether adding new entries to key-codes.js for each digit and a period is the right thing to do here, it'll certainly expand the pull request, though it's an easy addition. I've seen them called `NUM_0`=>`NUM_9` and `KEYPAD_0`=>`KEYPAD_9` elsewhere. If you do add new entries, can you ensure they are in alphabetical order?